modules/nf-core/sylph/profile/tests/main.nf.test

```diff
@@ -0,0 +1,79 @@
+nextflow_process {
+
+    script "../main.nf"
+    process "SYLPH_PROFILE"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "sylph"
+    tag "sylph/profile"
+
+    test("sarscov2 illumina single-end [fastq_gz]") {
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:true ], // meta map
+                            [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true) ]
+                        ]
+                input[1] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            def output_content = process.out.profile_out.get(0)
+
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions_single") },
+                { assert output_content.size() > 1 }
```

Comment:
Is there not a specific file or contents to test for here?

Reply:
Could you update this test so there's a specific file to test please?

Reply:
I updated the tests